samples/go/noms-merge/main.go

```diff
@@ -91,6 +98,60 @@ func nomsMerge() error {
 	}))
 }
 
+func resolve(aChange, bChange types.ValueChanged, a, b types.Value, path types.Path) (change types.ValueChanged, merged types.Value, ok bool) {
+	stringer := func(v types.Value) (s string, success bool) {
+		switch v := v.(type) {
+		case types.Bool, types.Number, types.String:
+			return fmt.Sprintf("%v", v), true
+		}
+		return "", false
+	}
+	left, lOk := stringer(a)
+	right, rOk := stringer(b)
+	if !lOk || !rOk {
+		return change, merged, false
+	}
+
+	// TODO: Handle removes as well.
+	fmt.Printf("\nConflict at: %s\n", path.String())
+	fmt.Printf("Left:  %s\nRight: %s\n\n", left, right)
+	var choice rune
+	for {
+		fmt.Println("Enter 'l' to accept the left value, 'r' to accept the right value, or 'm' to mash them together")
+		_, err := fmt.Scanf("%c\n", &choice)
+		d.PanicIfError(err)
+		switch choice {
+		case 'l', 'L':
+			return aChange, a, true
+		case 'r', 'R':
+			return bChange, b, true
+		case 'm', 'M':
+			if !a.Type().Equals(b.Type()) {
+				fmt.Printf("Sorry, can't smush a %s with a %s\n", a.Type().Describe(), b.Type().Describe())
+			}
+			switch a := a.(type) {
+			case types.Bool:
+				return aChange, types.Bool(bool(a) || bool(b.(types.Bool))), true
```

Comment:
Like, make it clear that value is relative to the key that was changed. I read it as an entirely new value.

Also see change type? does that make sense? Because the key is going to be constant, seems moot.